server/auth_test.go

```diff
@@ -229,3 +219,142 @@ func TestSwagger(t *testing.T) {
 		})
 	}
 }
+
+func TestPermissionsForSTTChecksPage(t *testing.T) {
+	ts := strconv.FormatInt(time.Now().Unix(), 10)
+	none := "none-" + ts
+	viewer := "viewer-" + ts
+	editor := "editor-" + ts
+	admin := "admin-" + ts
+
+	noneID := createUser(t, none)
+	defer deleteUser(t, noneID)
+
+	viewerID := createUserWithRole(t, viewer, "Viewer")
+	defer deleteUser(t, viewerID)
+
+	editorID := createUserWithRole(t, editor, "Editor")
+	defer deleteUser(t, editorID)
+
+	adminID := createUserWithRole(t, admin, "Admin")
+	defer deleteUser(t, adminID)
+
+	tests := []struct {
+		name       string
+		url        string
+		method     string
+		login      string
+		statusCode int
+	}{
+		{name: "settings-default", url: "/v1/Settings/Get", method: http.MethodPost, login: none, statusCode: http.StatusUnauthorized},
```

Comment:
why not just create another array and make for in for?

Reply:
To make it visually simple, one line - one test case.

Reply:
Do you guys really prefer `http.MethodPost` and `http.StatusUnauthorized` to `"POST"` and `401`? Uh!

Reply:
Well, yes. I always prefer predefined constants to string literals. I always thought that it is a common approach. 🤷‍♂️

Reply:
Will you use `const post = "POST"` and `const code401 = 401`? I think not. And that's almost the same thing.

Reply:
But I will 😐. I mean, if I need some value multiple times I usually put in constant.

Ok, I've got you point, will change.

Reply:
>  if I need some value multiple times I usually put in constant

String and number literals a constants too. We will use a named constant if name is better than the value, or if type is different, or if value is shared and should be updated in a single place – but any of that is not the case there.

Reply:
To put it in another way: you don't define `const n1 = 1` even if you use the value `1` several times :)

Reply:
I eliminate typos and able easily navigate with IDE through such constants. 
Seems like "holy war" topic)